plugins/module_utils/common.py

```diff
@@ -118,7 +118,7 @@ def configuration_digest(configuration):
     return digest
 
 
-def get_api_client(module=None):
+def get_api_client(module=None, **kwargs):
```

Comment:
Hi @Akasurde, can you explain why you need this? These parameters will be ignore.

Reply:
Inventory plugin calls `get_api_client` with args, which fails the inventory plugin.

Reply:
Yes, but these arguments will be ignored anyway. Maybe we should just remove them?

Reply:
No they are used.